src/components/CustomInstanceModal.vue

```diff
@@ -0,0 +1,82 @@
+<template>
+    <ModalComponent>
+        <h3 v-t="'Custom Instance'" class="font-bold my-4" />
+        <hr />
+        <div class="text-center">
+            <select v-model="selectedInstance">
+                <option
+                    v-for="(instance, pointer) in custominstance"
+                    :key="pointer"
+                    :value="pointer"
+                    v-text="instance.name"
+                />
+            </select>
+            <div class="flex justify-end">
+                <button @click="remove" v-t="'Remove'" />
+            </div>
+            <form class="children:pb-3">
+                <div>
+                    <input
+                        v-model="name"
+                        class="input w-full"
+                        type="text"
+                        :placeholder="'Name'"
+                        :aria-label="'Name'"
+                        v-on:keyup.enter="add"
+                    />
+                </div>
+                <div>
+                    <input
+                        v-model="url"
+                        class="input w-full"
+                        type="text"
+                        :placeholder="'Instance Api URL'"
+                        :aria-label="'Instance Api URL'"
+                        v-on:keyup.enter="add"
+                    />
+                </div>
+                <div class="flex justify-end">
+                    <button @click="add" v-t="'Add'" />
+                </div>
+            </form>
+        </div>
+    </ModalComponent>
+</template>
+
+<script>
+import ModalComponent from "./ModalComponent.vue";
+export default {
+    data() {
+        return {
+            custominstance: [],
+        };
+    },
+    mounted() {
+        if (this.testLocalStorage) {
+            if (localStorage.getItem("custominstance") === null) {
+                localStorage.setItem("custominstance", "[]");
+            }
+            this.custominstance = JSON.parse(localStorage.getItem("custominstance"));
+        }
+    },
+    methods: {
+        add() {
```

Comment:
When the inserted URL is invalid, the app will crash after the new instance got pushed. It would probably be a good idea to validate the inserted URL and showing an `alert` when it is invalid.